package-examples/hello-composite-pkg/README.md

```diff
@@ -0,0 +1,106 @@
+hello-composite-pkg
+==================================================
+
+# NAME
+
+  hello-composite-pkg
+
+# SYNOPSIS
+  
+  This is an example of a kpt composite package which has nested subpackages in it. 
+  Here are the steps to get, view, set and apply the package contents
+  
+  ### Fetch a remote package
```

Comment:
Instead of sections like this, I like to see a number set of steps:

1. Fetch a package
2. View the package contents
3. etc.